ajax/libs/alertify/package.json

```diff
@@ -0,0 +1,39 @@
+{
+  "name": "alertify",
+  "description": "An unobtrusive customizable JavaScript notification system",
+  "version": "0.3.11",
+  "homepage": "https://fabien-d.github.com/alertify.js/",
+  "author": {
+    "name": "Fabien Doiron",
+    "email": "fabien.doiron@gmail.com"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/fabien-d/alertify.js.git"
+  },
+  "keywords": [
+    "javascript",
+    "alert",
+    "notification"
+  ],
+  "filename": "alertify.min.js",
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/fabien-d/alertify.js.git",
+    "fileMap": [
+      {
+        "basePath": "lib",
```

Comment:
I think we also need another `basePath` for 0.4.0rc1. Thanks.

Reply:
Are we hosting alpha, beta and rc versions?

Reply:
Yes.